docs/rebuild_admin.rst

```diff
@@ -0,0 +1,407 @@
+
+From the *Admin Workstation*, open a terminal and copy the *Admin Workstation's* 
+SSH public key to the servers, substituting the appropriate values for the username
+and server IP addresses in the commands below and entering the admin account's 
+password when prompted:
+
+.. code:: sh
+
+  ssh-copy-id <admin-username>@<application-server-ip>
+  ssh-copy-id <admin-username>@<monitor-server-ip>
+  
+Next, create a file ``~/.ssh/config`` with contents as below, again substituting
+the appropriate values for your servers:
+
+.. code-block:: none
+ 
+  Host app
+    User <admin-username>
+    Hostname <application-server-ip>
+    ProxyCommand none
+
+  Host mon
+    User <admin-username>
+    Hostname <monitor-server-ip>
+    ProxyCommand none
+
+
+Finally, test direct SSH access from the terminal, using the commands ``ssh app`` and 
+``ssh mon``. It should be possible to connect without entering a password.
+
+Step 4: Retrieve SecureDrop configuration info from the servers
+===============================================================
+
+In addition to the account and networking information retrieved from the servers
+so far, you'll need to retrieve the following files and info:
+
+ - Tor hidden service URLs and tokens
+ - GPG *Submission Public Key* and OSSEC public key
+ - OSSEC alert configuration details
+ - (Optional) HTTPS configuration details
+
+Retrieve Onion hidden service info
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to connect to the Tor hidden services used by your instance, you will
+need to copy their details to the *Admin Workstation*. To do so, run the following
+commands from a Terminal window:
+
+.. code:: sh
+
+ cd ~/Persistent/securedrop/install_files/ansible-base
+ echo "HidServAuth $(ssh app sudo cat /var/lib/tor/services/journalist/hostname)" \
+    > app-journalist-aths
+ echo "$(ssh app sudo cat /var/lib/tor/services/journalist/hostname)" > app-source-ths
+
+If your instance was set up to use SSH over TOR, you'll also need to copy over the details 
+of the SSH proxy hidden services, by running the following commands:
+
+.. code:: sh
+
+ echo "HidServAuth $(ssh app sudo cat /var/lib/tor/services/ssh/hostname)" > app-ssh-aths
+ echo "HidServAuth $(ssh mon sudo cat /var/lib/tor/services/mon/hostname)" > mon-ssh-aths
+
+Retrieve GPG Public Keys 
+~~~~~~~~~~~~~~~~~~~~~~~~
+There are 2 GPG public keys required by the instance configuration, that you will need to 
+copy to the new *Admin Workstation*.
+
+To copy the *Submission Public Key*, download it from the *Source Interface* and import
+it locally using the following commands:
+
+.. code:: sh
+
+ cd ~/Persistent/securedrop/install_files/ansible-base
+ curl http://$(cat app-source-ths)/journalist-key > SecureDrop.asc
+ gpg --import SecureDrop.asc
+
+Validate that the imported key's fingerprint matches the one on your 
+SecureDrop install. You can do this by first running the command:
+
+.. code:: sh
+
+ gpg -k --fingerprint 
```

Comment:
This dumps a few different fingerprints, since Tails comes with a few keys preinstalled. Perhaps it would be worth verifying the fingerprint from the file before importing it. The short version to do that is `gpg --with-fingerprint SecureDrop.asc`, but that's ambiguous; the non-ambiguous version that worked for me on current Tails is `gpg --with-fingerprint --import-options import-show --dry-run --import SecureDrop.asc`.

Reply:
done.

Reply:
Leaving this open in case @emkll  or other reviewers have different suggestions for how to handle this part.